sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldUseBaseTypes.CSharp9.cs

```diff
@@ -0,0 +1,27 @@
+﻿public record A
+{
+    public A(A a) { }
+
+    public void Method() { }
+
+    public int Property { get; init; }
+}
+
+public record B : A
+{
+    public B(B b) : base(b) // Compliant - FN? only base class property is copied
```

Comment:
It looks like a FN to me. It can be changed to 'A' => FN. 
```suggestion
    public B(B b) : base(b) // FN only base class property is copied
```